lib/Mojolicious/Plugin/Webpack/Builder.pm

```diff
@@ -14,9 +14,9 @@ our $VERSION = $Mojolicious::Plugin::Webpack::VERSION || '0.01';
 
 has dependencies => sub {
   return {
-    core => [qw(webpack webpack-cli webpack-plugin-hash-output html-webpack-plugin)],
-    css  => [qw(css-loader mini-css-extract-plugin optimize-css-assets-webpack-plugin)],
-    js   => [qw(@babel/core @babel/preset-env babel-loader terser-webpack-plugin)],
+    core => [qw(webpack webpack-cli html-webpack-plugin@next)],
+    css  => [qw(css-loader mini-css-extract-plugin css-minimizer-webpack-plugin)],
+    js   => [qw(@babel/core @babel/preset-env babel-loader)],
```

Comment:
I don't understand these changes either.

Reply:
These are related to changes to support webpack 5, which has a lot of breaking changes. I added those because without them tests wouldn't pass.
They are big changes though, and I am not sure of the full implicances. I have a version running in my computer with these changes and running webpack 5, and seem to work fine, but other than that I can't tell.
I think the main issue here is to decide which webpack version will be the main supported version, and if the plugin should support older / newer versions.